### DISCUSSION TITLE
not all features are shown

### DISCUSSION THREAD

**Discussion by loreamat** [seerge/g-helper#1580] [Q&A]
Hi all I am having a problem with ghelper in that not all features are shown when opening the program. even if I open it as administrator. How can I solve this?

Asus Vivobook 15 rx571gt
![Senza titolo](https://github.com/seerge/g-helper/assets/62551151/acb619cc-f3cf-402c-8941-3d3a32e11963)

**Comment by seerge** [comment]
@loreamat only features supported by your device are shown.

You have a basic vivobook, that probably don't support dGPU disabling or MUX switch, etc.

**Reply by loreamat** [reply]
I remember the first time I opened it it at least showed the speed of the fans. now not even that. in any case, thank you
